config/public-api-v1.yml

```diff
@@ -31,31 +31,441 @@ paths:
+        qualifications:
+          type: string
+          description: |
+            The qualifications as an outcome of the course.
+          example: [ qts ]
+          enum:
+            - [ qts ]
+            - [ pgce ]
+            - [ pgde ]
+            - [ qts, pgce ]
+            - [ qts, pgde ]
+        recruitment_cycle_year:
+          type: string
+          description: The recruitment cycle that this course is available in.
+          example: 2020
+        required_qualifications:
+          type: string
+          format: markdown
+          description: |
+            The minimum academic qualifications needed for this course.
+          example: required_qualifications
+        required_qualifications_english:
+          type: string
+          description: "English GCSE requirements for applicants."
+          example: equivalence_test
+          enum:
+            - ""
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        required_qualifications_maths:
+          type: string
+          description: "Maths GCSE requirements for applicants."
+          example: equivalence_test
+          enum:
+            - ""
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        required_qualifications_science:
+          type: string
+          description: "Science GCSE requirements for applicants."
+          example: equivalence_test
+          enum:
+            - ""
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        running:
+          type: boolean
+          description: "Is the course currently running."
+          example: true
+        salary_details:
+          type: string
+          description: "Salary details about this course."
+          example: |
+            To be eligible for a place on the salaried course you must have the
+            support of a school who is willing to make contributions towards
+            your salary.
+        scholarship_amount:
+          type: integer
+          description: "The scholarship amount a candidate may be elligible for this course."
+          example: 17000
+        start_date:
+          type: string
+          format: date
+          description: "Date that the course starts."
+          example: "2020-09-01"
+        state:
+          type: string
+          description: |
+            The state of the course in the postgraduate teacher training system.
+          example: published
+          enum:
+            - empty
+            - rolled_over
+            - draft
+            - published
+            - published_with_unpublished_changes
+            - withdrawn
```

Comment:
```suggestion
            - withdrawn
            - terrible?
```

Reply:
I'd really like to be aiming, at least a little bit, for the data model we want, not the one we have. My thinking is that this is a relatively minor change which we could support with the existing data, but I can concede that this could have some issues. Why is it "terrible"?

Reply:
this `state` is just `terrible` :sob: 

we probably dont need to expose these

```
published_with_unpublished_changes #workflow no longer should be using this
empty # initial state
draft # initial state 

```